HybridLocalTemplate/index.html

```diff
@@ -2,10 +2,17 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, viewport-fit=cover">
     <link href="ratchet.css" rel="stylesheet">
```

Comment:
I think we should retire ratchet.css (it has not been updated in 4 years). But that's for another PR.